web/src/tools/testing/test-server/index.cjs

```diff
@@ -5,8 +5,21 @@ const port = 3000
 
 app.use(express.static(path.join(__dirname, '../../../../')))
 
+// for testing timeout error in web/src/test/manual/web/keyboard-errors
+const router = express.Router()
+router.get('/src/test/manual/web/keyboard-errors/timeout.js', async (req, res, next) => {
```

Comment:
timeout.js was deleted?

Reply:
yes, the file was deleted. But we still use the URL to trigger the test.

Reply:
Gotcha; can you add a comment accordingly to this test?